alias/README.md

```diff
@@ -0,0 +1,35 @@
+# Git kata: Aliases
+
+While working, we tend to write a lot of commands. This can get tedious, especially when the commands are rather long. Git allows you to create an alias for commands, dramatically shortening the time spent typing them.
```

Comment:
Should we add something about also not making typos?

Reply:
Also perhaps some thing along the lines of creating aliases for the different Git steps of our workflow could be useful.

Reply:
Good point on typos. I'll add that.
I'm not really sure I see what you mean with the aliases for different steps of the workflow. Can you give an example on what that would look like?

Reply:
My point was that if we have different steps in our workflow:
git start-work kata-123 = git checkout -b kata-123
or something like that. But I think we don't need this.